### PR TITLE
Update mainnet.md with Chainode Tech Archive Endpoints

### PR DESCRIPTION
**Pull request by Chainode**
It would also be good to add a WebSocket section - sometimes such a connection is preffered. 

Here is our endpoint for WebSocket: wss://celestia-ws.chainode.tech:33373/websocket

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

## Summary by CodeRabbit

- **New Features**
	- Added new RPC, API, gRPC, and WebSocket endpoints for Mainnet Beta participation.

### CODE REVIEW

**Comment by jcstein** [comment]
thanks for the PR and suggestion. would you like to add a websockets section in this PR?

File: nodes/mainnet.md [hunk 1]
```diff
@@ -162,6 +164,7 @@ to participate in Mainnet Beta:
 - `celestia.grpc.archives.validao.xyz:9090`
 - `gprc-archive.celestia.bitszn.com`
 - `celestia-grpc.f5nodes.com`
+- `celestia-grpc.chainode.tech:443`
```
Comment (jcstein) [inline nodes/mainnet.md:167]:
```suggestion
- `celestia-grpc.chainode.tech:443`

#### Websocket endpoints

- `wss://celestia-ws.chainode.tech:33373/websocket`
```

Comment (Chainode) [reply nodes/mainnet.md:167]: Sure, we can do this. Will commit the suggestion. Also, ideally the endpoints should have the exact protocol they are using http or https / ws or wss. Or there is the assumption that all endpoints are TLS encrypted.

File: nodes/mainnet.md [hunk 2]
```diff
@@ -140,6 +141,7 @@ to participate in Mainnet Beta:
 - `celestia.rest.archives.validao.xyz`
 - `api-archive.celestia.bitszn.com`
 - `celestia-api.f5nodes.com`
+- `celestia-api.chainode.tech`
```
[the remark this answers is not in the document]
Comment (Chainode) [reply nodes/mainnet.md:144]: Let me know if you want us to add also "https" in front of the endpoint. All our endpoints are TLS encrypted but as previously suggested - ideally all endpoints would exactly have the correct protocol in front.

**Review by jcstein — APPROVED** [review]

**Pull request merged**